package.json

```diff
@@ -16,7 +16,7 @@
     "build": "gulp build",
     "cover": "gulp coverage",
     "deploy": "npm run lint && npm run test && scripts/deploy.sh",
-    "lint": "gulp jshint",
+    "lint": "gulp jshint && gulp tslint",
```

Comment:
are we leaving any js code behind.  If not, why still running jshint?

Also a minor point that can be done later, if we can just run tslint directly, better run it directly?